cmd/download_test.go

```diff
@@ -144,7 +144,7 @@ func TestDownload(t *testing.T) {
 
 		dir := filepath.Join(targetDir, "bogus-track", "bogus-exercise")
 		b, err := ioutil.ReadFile(workspace.NewExerciseFromDir(dir).MetadataFilepath())
-		var s workspace.Solution
+		var s workspace.ExerciseMetadata
```

Comment:
I think `s` is probably the wrong variable name now that the type has been renamed. How would you feel about `metadata` to match the variable names you've used in the matching command file? Since it's tests, we could also do something shorter (`em`? `meta`?). I don't know. I like the consistency of `metadata`.